# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/03 09:08:30+01:00 perex@suse.cz 
#   [ALSA] ifdef typos: sound_isa_cs423x_cs4231_lib.c
#   
#   CS4231 driver
#   Funny typo.
#   
#   Acked-by: Randy Dunlap <rddunlap@osdl.org>
#   
#   Signed-off-by: Domen Puncer <domen@coderock.org>
#   Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
#   Signed-off-by: Takashi Iwai <tiwai@suse.de>
# 
# sound/isa/cs423x/cs4231_lib.c
#   2004/11/30 07:33:24+01:00 perex@suse.cz +1 -1
#   [ALSA] ifdef typos: sound_isa_cs423x_cs4231_lib.c
#   
#   D:2004/11/30 14:33:24
#   C:CS4231 driver
#   F:isa/cs423x/cs4231_lib.c:1.45->1.46 
#   L:Funny typo.
#   L:
#   L:Acked-by: Randy Dunlap <rddunlap@osdl.org>
#   Signed-off-by: Domen Puncer <domen@coderock.org>
#   Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
#   Signed-off-by: Takashi Iwai <tiwai@suse.de>
# 
diff -Nru a/sound/isa/cs423x/cs4231_lib.c b/sound/isa/cs423x/cs4231_lib.c
--- a/sound/isa/cs423x/cs4231_lib.c	2005-01-22 23:37:19 -08:00
+++ b/sound/isa/cs423x/cs4231_lib.c	2005-01-22 23:37:19 -08:00
@@ -685,7 +685,7 @@
 
 	snd_cs4231_mce_down(chip);
 
-#ifdef SNDRV_DEBUGq_MCE
+#ifdef SNDRV_DEBUG_MCE
 	snd_printk("init: (1)\n");
 #endif
 	snd_cs4231_mce_up(chip);
